frontend/src/components/ListDetail.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { addItem } from "../http/lists";
+
+export const ListDetail = ({selectedList, setSelectedList}) => {
+    const [messageState, setMessageState] = useState('')
+    const [newItemName, setNewItemName] = useState('')
+
+    const handleAddItem = async(event) => {
```

Comment:
will add in future pr

Reply:
done

Reply:
I think in lieu of: 
```
// Takes the value from the input at the bottom of the list
// and adds it to the list of items within a specific list
// checks to see if the key pressed was the enter key 
// we display a message below the input if successful or error
```
This would be more useful:
```
// tries to add an item to the selected list when enter is pressed.
```

Reply:
fixed